charts/render.py

```diff
@@ -18,10 +18,13 @@
   `app.kubernetes.io/component` fields
 '''
 
+import re
```

Comment:
(unrelated to this line, for the commit)

This commit is required before rendering the chart, so let's move it before.
Also, the commit message could be updated to:
- include a reference to the Dex deployment using charts issue (so we know that it's linked somehow)
- detail why multi-line strings are an issue in rendered charts (it's not a problem in YAML per-se, but doesn't work well when Jinja "commands" are split)